### PR TITLE
Updated documentation for SeLion session sharing feature

### PR DESCRIPTION
**Pull request by sundaramrajendran**

### CODE REVIEW

File: html/webdriversessionsharing.html [hunk 1]
```diff
@@ -11,44 +11,41 @@ <h1 class="page-header">Sharing WebDriver Sessions</h1>
 
   <h2>How to Share WebDriver Sessions Between Tests</h2>
```
Comment (squash-merge) [inline html/webdriversessionsharing.html]: `Steps to Share WebDriver Sessions Between Tests` may to more appropriate.

**Comment by squash-merge** [comment]
Other review comments I have are submitted to @sundaramrajendran via https://github.com/sundaramrajendran/SeLion/pull/1

File: html/webdriversessionsharing.html [hunk 2]
```diff
@@ -11,44 +11,41 @@ <h1 class="page-header">Sharing WebDriver Sessions</h1>
...
+
+      <pre data-src="../sample-files/webdriversessionsharing-sample4.java"></pre>
       Web sessions can be shared to <var>n</var> <code>@Test</code> methods as long as test cases are running sequentially.
       Here is an example where the session is shared across three <code>@Test</code> methods.
 
-      <pre data-src="../sample-files/webdriversessionsharing-sample4.java"></pre></li>
+      <pre data-src="../sample-files/webdriversessionsharing-sample5.java"></pre></li>
   </ol>
 
   <h2>Tips for Using Web Session Sharing Effectively</h2>
```
Comment (fakrudeen78) [inline html/webdriversessionsharing.html]: I think, Now this tips section is not needed

File: html/webdriversessionsharing.html [hunk 3]
```diff
@@ -11,44 +11,41 @@ <h1 class="page-header">Sharing WebDriver Sessions</h1>
...
+    <li>To pick up the same test session in a different test, annotate the test with <code>@Test</code>
+        and use the <code>priority</code> greater than defined in the master test. This will reuse the same session in this test 
+        instead of creating a new session. Since these tests order of execution controlled by TestNG priority.
+        <p class="alert-info">Note: It's good practice to keep these tests in the same group.</p>
 
       <pre data-src="../sample-files/webdriversessionsharing-sample3.java"></pre>
+      Here is a complete example that shows how a typical test would look:
+
+      <pre data-src="../sample-files/webdriversessionsharing-sample4.java"></pre>
       Web sessions can be shared to <var>n</var> <code>@Test</code> methods as long as test cases are running sequentially.
```
Comment (fakrudeen78) [inline html/webdriversessionsharing.html]: Instead of telling Web Session sharing. Lets explain in general way ( Session Sharing)

**Comment by sundaramrajendran** [comment]
Addressed all the review comments

**Comment by mach6** [comment]
This is merged.

**Pull request closed without merging**